op-bindings/gen/main.go

```diff
@@ -97,18 +130,7 @@ func main() {
 
 		deployedSourceMap := ""
 		if _, ok := sourceMapsSet[name]; ok {
-			// directory has .sol extension
```

Comment:
My guess is that this got out of date due to `err != nil` and this is resulting in a diff for the preimage oracle source map